examples/integrations/hamilton/README.md

```diff
@@ -0,0 +1,7 @@
+# Examples using the Hamilton integration
+
+Burr has some syntactic sugar for integrating with Hamilton. Find examples of them here.
```

Comment:
We're not using any of the syntactic sugar, might as well remove this (or fix it up to use it).